test/unit/components/useIntl.js

```diff
@@ -25,10 +25,9 @@ describe('useIntl() hook', () => {
         <FunctionComponent spy={spy} />
       </IntlProvider>
     );
-    const intlProvider = rendered.find(IntlProvider).childAt(0);
 
-    expect(spy).toHaveBeenCalledWith(
-      intlProvider.instance().getContext()
-    );
+    const providedIntl = rendered.find(IntlProvider).childAt(0).instance().getContext();
+    const expectedHookReturns = [providedIntl.formatMessage, providedIntl];
```

Comment:
You might as well change the name from `providedIntl` to plain `intl` and inline the expect value, to make the code easier to understand and reflect the tested code in an easier to grasp way.
```js
const intl = rendered.find(IntlProvider).childAt(0).instance().getContext();
expect(spy).toHaveBeenCalledWith([ intl.formatMessage, intl ]);
```

Reply:
@DragonRaider5, Nice! All green and look good to me now =)